check/check_test.go

```diff
@@ -69,6 +69,31 @@ func TestCheck_Run(t *testing.T) {
 			},
 			Expected: PASS,
 		},
+		{
+			name: "Scored checks that pass should PASS when config file is not present",
```

Comment:
This test case failed. could you please fix this test? Thanks.

Reply:
Strange, the same testcases is running fine in my local
```
➜  kube-bench git:(deven-bugfix-file-not-found) ✗ make tests
GO111MODULE=on go test -vet all -short -race -timeout 30s -coverprofile=coverage.txt -covermode=atomic ./...
?   	github.com/aquasecurity/kube-bench	[no test files]
ok  	github.com/aquasecurity/kube-bench/check	1.397s	coverage: 91.4% of statements
ok  	github.com/aquasecurity/kube-bench/cmd	11.518s	coverage: 56.5% of statements
?   	github.com/aquasecurity/kube-bench/internal/findings	[no test files]
```